exercise1.js

```diff
@@ -81,22 +153,71 @@ const nums = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
 // value in the nums array, and every 3rd number, i.e. 0, 3, 6,
 // and 9: the zeroth, third, sixth, and ninth values.
 
-//your code...
+for(let x = 0; x<nums.length; x++){
+    if(nums[0] === nums[x]){
+        console.log("The zeroth");
+    }
+    else if(nums[3] === nums[x]){
+        console.log("The third")
+    }
+    else if(nums[6] === nums[x]){
+        console.log("The sixth")
+    }
+    else if(nums[9] === nums[x]){
+        console.log("The ninth")
+    }
+}
 
 
 
 // Problem 9:
 const foodArray = [ 'potatoes', 'tamales', 'lemon','strawberries','chocolate', 'pudding', {school : 'TEKcamp'} ];
 //access the value of the last element of the array and set it to a variable called school.  print the school variable to the console.
+let school = foodArray[foodArray.length-1];
+console.log(school);
 
 const adjectiveArray = [ 'salty', 'spicy', 'sour', 'sweet', 'rich','creamy','amazing'];
 // Using both the foodArray and the adjectiveArray, make a
 // "for" loop that console.log()'s a sentence for each
 // corresponding value in the arrays. Add the word "is" or "are" depending on if the food is singular or plural.  "Potatoes are salty", "Lemon is sour".
 
+for(let x = 0; x<foodArray.length; x++){
+    if(foodArray[x][foodArray[x].length-1] == 's'){
+        console.log(foodArray[x]+" are "+ adjectiveArray[x]);
+    }
+    else{
+        console.log(foodArray[x]+ " is "+ adjectiveArray[x]);
+    }
+}
+
```

Comment:
nice clean way of doing it.  easy to follow

only issue is, you aren't pulling the value out of the last object in the array.  so the output doesn't look correct.